src/main/java/com/napier/sem/DAO.java

```diff
@@ -107,12 +107,12 @@ public boolean queryInvalid(String queryName,
     {
         if (whereCondition == null || n < 1)
         {
-            System.out.printf("%s - invalid query condition", queryName);
+            System.out.printf("%s - invalid query condition. ", queryName);
```

Comment:
There was some output from one of the tests where this was parsed before another string so added this just incase its a possible output while running the console app.